deploy/chart/values.yaml

```diff
@@ -48,4 +48,4 @@ tolerations:
 # LINODE_HOSTNAME_ONLY_INGRESS type bool is supported
 # env:
   # - name: EXAMPLE_ENV_VAR
-  #   value: "true"
\ No newline at end of file
+  #   value: "true"
```

Comment:
there was a problem with the ccm chart, we'd need to set `image.tag` explicitly for it to actually install the version we want. the chart defaults to `.appVersion` only if `image.tag` is not set, and we set it to `latest` in values. could comment it out in the values.yaml for a quick fix